src/api/schemas/log_schema.py

```diff
@@ -22,3 +22,6 @@ class Meta:
     args = fields.Dict()
     json = fields.Field(allow_none=True)
     error = fields.Str()
+    application_name = fields.Str(required=False)
```

Comment:
It might be nice to sort these fields if order doesn't matter.

Reply:
resolved with [dbc3986](https://github.com/cisagov/domain-manager-api/pull/515/commits/dbc3986ca051ee55957f99a5bf85ffaefc30caf7)